parsers/s01-parse/baudneo/gotify-logs.yaml

```diff
@@ -0,0 +1,24 @@
+#filter: '1 == 1'  # For hub tests
+filter: evt.Parsed.program == "gotify"  # Production
+#debug: true
+onsuccess: next_stage
+name: baudneo/gotify-logs
+description: parser for Gotify server
+pattern_syntax:
+  GOTIFY_DATE: '%{YEAR}/%{MONTHNUM}/%{NUMBER}%{SPACE}[-]%{SPACE}%{TIME}'
+    GOTIFY_SEPERATOR: '%{SPACE}\|%{SPACE}'
+    #  GOTIFY_DATE: \d*/\d*/\d*\s[\-]\s.*
+    #  GOTIFY_SEPERATOR: \s*\|\s*
+    GOTIFY_401: '^\[GIN\]\s*%{GOTIFY_DATE}%{GOTIFY_SEPERATOR}401%{GOTIFY_SEPERATOR}%{DATA:request_time_took}%{GOTIFY_SEPERATOR}%{IP:source_ip}%{GOTIFY_SEPERATOR}%{WORD:request_type}%{SPACE}"%{DATA:endpoint}"'
```

Comment:
Seems the indentation is not correct ?

Reply:
I have corrected the indentation and matched the static Meta.source_ip to the GROK pattern.